cmd/resultsapi/queues.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	common "github.com/GSA-TTS/jemison/internal/common"
+	"github.com/GSA-TTS/jemison/internal/env"
+	"github.com/GSA-TTS/jemison/internal/queueing"
+	"github.com/jackc/pgx/v5"
+	"github.com/jackc/pgx/v5/pgxpool"
+	"github.com/riverqueue/river"
+	"github.com/riverqueue/river/riverdriver/riverpgxv5"
+	"go.uber.org/zap"
+)
+
+// The work client, doing the work of `ResultsApi`.
+var ResultsAPIPool *pgxpool.Pool
+var ResultsAPIClient *river.Client[pgx.Tx]
+
+type ResultsAPIWorker struct {
+	river.WorkerDefaults[common.ResultsAPIArgs]
+}
+
+func InitializeQueues() {
```

Comment:
Food for thought... perhaps we do *not* set up a client at this time. No harm in doing so, but we don't expect messages *in*. :shrug: